packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -628,6 +628,7 @@ export async function activate(context: vscode.ExtensionContext) {
     EmptyParametersGatherer,
     getDefaultUsernameOrAlias,
     getUserId,
+    getRootWorkspacePath,
```

Comment:
I originally wrote almost the same function in our extension but decided it would be more dry to reuse the one in core.

Reply:
We are going to move this function to utils module soon, so it should be fine if you use a copy of that.

Reply:
The goal is not to depend on the core extension

Reply:
ok, I can refactor a bit to not depend on the core extension.

Reply:
@xyc  So to clarify are you saying I should write my own getRootWorkspacePath function in our extension, or that it is fine now to use core but we should use the Utils module when its created?

It would be nice to not duplicate functions but in this case maybe best if we just have our own so we don't depend on core or have to make changes later.

Reply:
I think if you copied the `getRootWorkspacePath` it would be fine. Otherwise, we'd still need to remove the exports in core here. In terms of amount of changes we need to make it's similar for both approaches, but I would like us not to export more utilities methods like this from core.

Reply:
got it, I'll fix it =)

Reply:
fixed in the `fix(soql): remove dependency from core extension in fileservice` commit.